evap/staff/templates/staff_evaluation_person_management.html

```diff
@@ -110,3 +114,9 @@ <h6 class="card-subtitle mb-2 text-muted">{% trans 'To CSV file' %}</h6>
         </div>
     </div>
 {% endblock %}
+
+{% block additional_javascript %}
+    <script type="text/javascript">
+        {% include "copy-to-clipboard.js"%}
```

Comment:
missing space before closing `%}` and please use `'` instead of `"` for the include